res/themes/light-custom/css/_custom.pcss

```diff
@@ -145,3 +139,23 @@ $menu-selected-color: var(--menu-selected-color, $menu-selected-color);
 $pill-bg-color: var(--other-user-pill-bg-color, $pill-bg-color);
 $pill-hover-bg-color: var(--other-user-pill-bg-color, $pill-hover-bg-color);
 $icon-button-color: var(--icon-button-color, $icon-button-color);
+
+/**
+ * Variation of the accent color.
+ * Generate this colour scale using LeonardoColor
+ * TODO: Add link
```

Comment:
Pending TODO, is this going to be done as part of this PR?

Reply:
No, it won't be

Reply:
Then please open an issue to track the work and link it here :D

Reply:
https://github.com/vector-im/compound/issues/158